routes/users.js

```diff
@@ -822,7 +819,7 @@ router.post("/signup", async (req, res) => {
 // Register Handle for web
 router.post("/register", async (req, res) => {
   // check required fields
-  const { name, email, password, location } = req.body;
+  const { name, email, password, location,likes, dislikes,factors} = req.body;
   if (!name || !email || !password || !location) {
```

Comment:
When FE is creating new users, they don't need to send {likes, dislikes, factors} - it's useless code.
{likes, dislikes, factors} will be empty from the beginning and will be filled after another endpoints

Reply:
I  had removed those items before from req.body. I don't know how it happens again. Anyway, I had fixed it.

Reply:
I understand what you are trying to say. I have removed items from req.body. while fixing the error it updated. Anyway fixed it